cli-files/go-ls/cmd/root_test.go

```diff
@@ -0,0 +1,90 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestGolsCmdFunc(t *testing.T) {
+	t.Run("No path provided", func(t *testing.T) {
```

Comment:
All of these tests seem to do pretty much the same thing - can you think of a way to make them less repetitive?

One convention in Go is to do something called table driven testing, which may help.

Reply:
Got it, updated